### PR TITLE
Conformance section examples are now single items in each statement

### PR DESCRIPTION
**Pull request by GeorgeKerscher**
The changes makes the conformance much cleaner.

In the conformance section, each statement is in a paragraph with a single entry. This assumes a placeholder will be used to extract the metadata, such as the date into the item.

I expect conformance to EPUB 1.0 or 1.1, the WCAG number, and the level will be modified in the techniques. This will require changes in this section after the techniques are finished.

### CODE REVIEW

**Review by gautierchomel — APPROVED** [review]

File: a11y-meta-display-guide/2.0/draft/guidelines/index.html [hunk 1]
```diff
@@ -981,18 +968,14 @@ <h4>Examples</h4>
...
+							publication was certified by</span>
 						Foo's Accessibility Testing
 					</p>
 					<p>
 						<span
 							data-localization-id="conformance-certifier-credentials-prefix"
-							data-localization-mode="compact">with a
-							credential of</span> "Enterprise
-						Accessibility Rating."
+							data-localization-mode="compact">The certifier 's credential is</span> "Enterprise
```
Comment (clapierre) [inline a11y-meta-display-guide/2.0/draft/guidelines/index.html:977]: same extra space here before '

File: a11y-meta-display-guide/2.0/draft/guidelines/index.html [hunk 2]
```diff
@@ -1014,17 +997,11 @@ <h4>Examples</h4>
...
-							data-localization-id="conformance-certifier-prefix"
-							data-localization-mode="descriptive">by</span>
-						Foo's Accessibility Testing <span
-							data-localization-id="conformance-certifier-credentials-prefix">with
-							a credential of</span> "Enterprise
-						Accessibility Rating" <span
+							publication was certified on</span>
+					2021-09-07</p>
+						<p><span data-localization-id="conformance-certifier-credentials-prefix">The certifier 's credential is</span> "Enterprise
+						Accessibility Rating" </p>
```
Comment (clapierre) [inline a11y-meta-display-guide/2.0/draft/guidelines/index.html:1003]: extra space before '

File: a11y-meta-display-guide/2.0/draft/guidelines/index.html [hunk 3]
```diff
@@ -1049,17 +1026,19 @@ <h4>Examples</h4>
...
-						specification.)</p>
+						specification</p>
 					<p> The publication was certified
-						on 2021-09-07 by Foo's Accessibility Testing
-						with a credential of "Enterprise Accessibility
-						Rating." For more information refer to the <a
+						on 2021-09-07 </p>
+<p>
+						The certifier 's credential is"Enterprise Accessibility
+						Rating" </p>
```
Comment (clapierre) [inline a11y-meta-display-guide/2.0/draft/guidelines/index.html:1040]: seems like all "certifier 's" has the extra space

**Review by clapierre — CHANGES_REQUESTED** [review]
once the extra spaces are resolved, this looks good.

**Pull request merged**